collector/mongodb_collector_test.go

```diff
@@ -55,22 +55,31 @@ func TestCollector(t *testing.T) {
 		close(metricCh)
 	}()
 
-	var descs int
-	for range descCh {
-		descs++
+	var descriptors = make(map[string]struct{})
+	var descriptorsCount int
+	for d := range descCh {
+		descriptors[d.String()] = struct{}{}
+		descriptorsCount++
 	}
 
-	var metrics int
+	var metricsCount int
 	var versionInfoFound bool
 	for m := range metricCh {
+
+		if _, ok := descriptors[m.Desc().String()]; ok {
+			delete(descriptors, m.Desc().String())
+		}
```

Comment:
`assert.True(t, ok)` with a good message.

Reply:
I'll print in after the cycle as 2 metrics (with different labels) have same descriptor. and if I do it here we will duplicate output when

Reply:
Cool. Is it a real error message? That one seems easy to fix.

Reply:
> Cool. Is it a real error message? That one seems easy to fix.

For now it's just some imitation of an error. But when it happens in CI, it will be real and we can fix it.